phpstan.neon

```diff
@@ -2,8 +2,9 @@ parameters:
     tmpDir: /tmp/tools-cache/.phpstan
     parallel:
         processTimeout: 300.0
-    checkMissingIterableValueType: false
     ignoreErrors:
+        -
+            identifier: missingType.iterableValue
```

Comment:
:confused: why?

Reply:
this is new way, how to ignore errors in PHPStan from 1.11: https://phpstan.org/error-identifiers